contracts/hyrun/src/contract.rs

```diff
@@ -53,11 +53,25 @@ where
     };
     println!("Fetched current state: {:?}", initial_state);
 
+    let contract_input = build_contract_input(initial_state);
+
+    println!("{}", "-".repeat(20));
+    println!("Checking transition for {contract_name}...");
+    let execute_info = execute(contract_name, &contract_input);
+    let output = execute_info.journal.decode::<HyleOutput>().unwrap();
+    if !output.success {
```

Comment:
Hum, si on ne prouve pas lorsque success est à false, je me demande si on a seulement besoin du `success` dans HyleOutput :thinking: 
Je ne me souviens plus de pourquoi on l'avait mis. Enfin si, on l'avait mis pour être en mesure de prouver un échec, mais dans quel cas aurait-on besoin de prouver un failure ? Sachant que la ProofTx d'une failure ne sera jamais process pour settle...? 
@wraitii si jamais tu te souviens

Reply:
pour pouvoir fail rapidement le settlement

Reply:
Ici on est dans l'utilitaire pour générer des transactions, si on peut s'éviter d'envoyer des transactions inutiles autant le faire... Sur eth si tu craft une tx en sachant que t'as pas mis assez de gas, tu vas pas la broadcaster...

Reply:
ça peut être pratique de faire fail un settlement plutôt que de rester "in limbo" dans certains cas, mais je suis pas opposé à pas le faire là